vultr/resource_vultr_kubernetes.go

```diff
@@ -80,6 +80,20 @@ func resourceVultrKubernetes() *schema.Resource {
 				Computed:    true,
 				Sensitive:   true,
 			},
+			"cluster_ca_certificate": {
+				Type:     schema.TypeString,
+				Computed: true,
```

Comment:
We should probably default these to `Sensitive: true`, so they aren't printed in the output unless explicit, like `kube_config`